### PR TITLE
#486 - Bug Uploaded records for institution file upload

### PR DESCRIPTION
**Pull request by guru-aot**
Uploaded records bug for institution file upload

### CODE REVIEW

**Review by ann-aot — COMMENTED** [review]
👍

**Review by ann-aot — APPROVED** [review]
👍

**Review by andrewsignori-aot — APPROVED** [review]
👍

File: sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece.processing.service.ts [hunk 1]
```diff
@@ -95,9 +96,12 @@ export class ECEProcessingService {
         fileInfo.filePath,
       );
       this.logger.log("Content uploaded.");
+      const eceFileFooter = fileContent[
```
Comment (dheepak-aot) [inline sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece.processing.service.ts:99]: `const [eceFileFooter]= fileContent.slice(-1);` ?

**Review by dheepak-aot — APPROVED** [review]
Approving with optional comment.

**Pull request merged**